e3sm_warehouse/resources/warehouse_config.yaml

```diff
@@ -1,4 +1,4 @@
-DEFAULT_WAREHOUSE_PATH: /p/user_pub/e3sm/warehouse/
```

Comment:
Should we rename this directory to align with the repo and package?

Reply:
No, I'd want to avoid that.  It already has e3sm/warehouse in the path, and e3sm/e3sm_warehouse would be redundant.  Also, MANY scripts and config files rely upon it - that would be a lot of editing and testing.

Reply:
Got it, that's what I was thinking too. I will revert my "find and replace" string changes that affected these paths.